docs/src/Checkbox.doc.js

```diff
@@ -85,64 +93,35 @@ card(
     name="Example: Accessibility"
     defaultCode={`
 function CheckboxExample() {
-  const [checked, setChecked] = React.useState(true);
```

Comment:
@AlbertCarreras Can we leave 1 example (medium) in here instead of also adding small?
We always want the first example to be easily copy/paste-able and the default (medium in this case).

Reply:
Oh, ok.

Reply:
DONE